backend/core/admin.py

```diff
@@ -58,9 +59,29 @@ def deactivate_team_members(self, request, queryset: object):
         ),
     )
     inlines = [UserSetInline]
-    actions = [deactivate_team_members]
+    actions = ['deactivate_team_members', 'bulk_regenerate_cards_for_members']
     ordering = ["name"]
+    
+    def deactivate_team_members(self, request, queryset: object):
+        for team in queryset:
+            for team_member in team.active_users:
+                team_member.active = False
+                team_member.save()
 
+    def bulk_regenerate_cards_for_members(self, request, queryset: object):
+        if request.POST.get('post', None):
+            for team in queryset:
+                bulk_regenerate_cards_for_team(team)
```

Comment:
This would be a very long running request. It should be handled by the dramatiq worker. Take a look at how the current delete and recreate cards api works. 

I would append one task for each user who needs to be regenerated.

Reply:
So something like:

```
from long_running_request_actors import (
                delete_and_recreate_user_cards as actor,
            )

for user_id in active_memer_ids:
    actor.send_with_options(kwargs={"user_id": user.id})

```

Reply:
What would you recommend, spawning `delete_and_recreate_user_cards` actor for each user in a team in a new actor, say `bulk_regenerate_cards_for_members`, or running that separate actor, doing all the regeneration for all users at once

Reply:
Something like:

```python
@actor(max_retries=3)
def bulk_regenerate_cards_for_team(team):
    for user in team.users.filter(active=True):
        delete_and_recreate_user_cards.send_with_options(kwargs={"user_id": user.id})
```

Reply:
Better to do it the way I suggested. It would be faster because regens can happen in parallel. Also if there are any issues with regenning one user's board then that wont totally derail the process for all subsequent users.